node/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # syntax=docker.io/docker/dockerfile:1.3@sha256:42399d4635eddd7a9b8a24be879d2f9a930d0ed040a61324cfdf59ef1357b3b2
-FROM --platform=linux/amd64 docker.io/golang:1.20.5@sha256:4b1fc02d16fca272e5e6e6adc98396219b43ef663a377eef4a97e881d364393f AS build
+FROM --platform=linux/amd64 docker.io/golang:1.20.7-bullseye@sha256:74b09b3b6fa5aa542df8ef974cb745eb477be72f6fcf821517fb410aff532b00 AS build
```

Comment:
happened to notice that the target is "bullseye" is that a new pattern for golang releases?  Just caught my eye and wanted to make sure it wasn't a totally different branch target.

Reply:
golang upgraded the image to `bookworm` which isn't supported by the CI runners yet. Bullseye has support til 2026.

Reply:
it's because there's now multiple debian bases. this refers to the debian version (bullseye is the LTS which was previously being used).

Reply:
@tbjump and I spent a bit of time on this together. The issue is that the newer version of debian (bookworm) uses a glibc that has an incompatibility with the seccomp policy the CI runner is using.

```
$ docker run -it docker.io/golang:1.20.7 cat /etc/os-release  | grep ^PRETTY
PRETTY_NAME="Debian GNU/Linux 12 (bookworm)"
```

Bullseye is supported until June 2026, so we can fix things by then.